servo-build-dependencies/init.sls

```diff
@@ -10,6 +10,7 @@ servo-dependencies:
       - automake
       - pkg-config
       - openssl
+      - freetype2
```

Comment:
We use Homebrew for installing packages on OS X, which calls this package `freetype`.